packages/cyberstorm/src/components/Switch.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropsWithChildren, useCallback } from "react";
+import styles from "./componentStyles/Switch.module.css";
+import * as RadixSwitch from "@radix-ui/react-switch";
+
+export interface SwitchProps {
+  state: boolean;
+  switchCallback?: (state: boolean) => void;
```

Comment:
This should probably be called something along the lines of `onChange` or whatever is the most standard. Really, the props here should probably emulate the props that exist in the normal checkbox input in React so that this component is easy to use in forms if necessary.